crates/common_utils/src/macros.rs

```diff
@@ -95,3 +95,14 @@ macro_rules! collect_missing_value_keys {
         }
     };
 }
+
+#[macro_export]
+macro_rules! get_formatted_date_time {
+    ($date_format:tt) => {
+        {
+        let format=time::macros::format_description!($date_format);
+        time::OffsetDateTime::now_utc()
+            .format(&format)
+            .change_context(errors::ConnectorError::InvalidDateFormat)} 
```

Comment:
If this macro is tied to a specific error type, in this case, `ConnectorError`, I'd prefer it being defined in the connector utils module rather than in the `common_utils` crate.

Reply:
sure